Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+requests = "*"
```

Comment:
I'm not familiar with Pipfile. Why is `requests = *` here but in `requirements.txt` something different? :-)

Reply:
`Pipfile` is what `pipenv` uses.
`requirements.txt` doesn't take a wildcard so it needs to be a version number. I've only included it as a fallback.
Would you like `Pipfile` to have the same version constraint as `requirements.txt`?

Reply:
Thanks! No, that's not necessary. This PR looks good to me. 👍